src/client/src/pages/detail/labeling/labelingFileSelector.tsx

```diff
@@ -0,0 +1,116 @@
+import { Box, Button, CircularProgress, Divider, Stack, Typography } from "@mui/material";
+import { File as FileInterface, maxFileSizeKB } from "../../../api/file/fileInterfaces.ts";
+import { File as FileIcon } from "lucide-react";
+import { AddButton } from "../../../components/buttons/buttons.tsx";
+import { FC, useCallback, useContext, useRef } from "react";
+import { useTranslation } from "react-i18next";
+import { FileRejection, useDropzone } from "react-dropzone";
+import { AlertContext } from "../../../components/alert/alertContext.tsx";
+import { labelingFileFormat } from "./labelingInterfaces.tsx";
+
+interface LabelingFileSelectorProps {
+  isLoadingFiles: boolean;
+  files?: FileInterface[];
+  setSelectedFile: (file: FileInterface) => void;
+  addFile: (file: File) => void;
+}
+
+const LabelingFileSelector: FC<LabelingFileSelectorProps> = ({ isLoadingFiles, files, setSelectedFile, addFile }) => {
+  const { t } = useTranslation();
+  const fileInputRef = useRef<HTMLInputElement>(null);
+  const { showAlert } = useContext(AlertContext);
+
+  const onDrop = useCallback(
+    (acceptedFiles: File[], fileRejections: FileRejection[]) => {
+      if (fileRejections.length > 0) {
+        const errorCode = fileRejections[0].errors[0].code;
+        const errorMessages: { [key: string]: string } = {
+          "file-invalid-type": t("fileInvalidType"),
+          "too-many-files": t("fileTooMany"),
+          "file-too-large": t("fileMaxSizeExceeded"),
+        };
+        showAlert(errorMessages[errorCode] || fileRejections[0].errors[0].message, "error");
+      } else {
+        addFile(acceptedFiles[0]);
+      }
+    },
+    [addFile, showAlert, t],
+  );
+
+  const { getRootProps, getInputProps } = useDropzone({
+    onDrop,
+    maxFiles: 1,
+    maxSize: maxFileSizeKB,
+    accept: { [labelingFileFormat]: [] },
+    noDrag: false,
+    noClick: true,
+  });
+
+  return (
+    <Box
+      sx={{ padding: "84px 50px", height: "100%", width: "100%", cursor: "pointer" }}
+      {...getRootProps()}
+      onDragOver={e => {
+        e.stopPropagation();
+        e.preventDefault();
+        e.dataTransfer.dropEffect = "copy";
+      }}>
+      <input {...getInputProps()} data-cy="labeling-file-dropzone" ref={fileInputRef} />
+      <Stack
+        sx={{
+          justifyContent: "center",
+          alignItems: "center",
+          height: "100%",
+          width: "100%",
+          border: "2px solid transparent",
+          borderImage: "repeating-linear-gradient(45deg, #ffffff, #ffffff 10px, transparent 10px, transparent 20px) 1",
```

Comment:
```suggestion
          borderRadius: "24px",
          backgroundImage: `url("data:image/svg+xml,%3Csvg width='100%25' height='100%25' xmlns='http://www.w3.org/2000/svg'%3E%3Crect width='100%25' height='100%25' fill='none' rx='24' ry='24' stroke='%23fff' stroke-width='4' stroke-dasharray='6%2C 14' stroke-dashoffset='20' stroke-linecap='square'/%3E%3C/svg%3E")`,
```
von hier: https://kovart.github.io/dashed-border-generator/
bisschen hacky, ich überlasse es dir, ob du es übernimmst, finde die Version ohne Kurve sonst auch ok.

Reply:
Mit dem code sah's bei mir gut aus

Reply:
![image](https://github.com/user-attachments/assets/5c1d8a83-2285-4bf4-b6df-e350217755d3)
 Sieht nicht bei jeder Zoomstufe perfekt aus, aber ich finds ok . Aber wie gesagt für mich beides io